### ISSUE TITLE
photoprism invalid originals path

### ISSUE THREAD

**Issue by NormannK** [tteck/Proxmox#3978]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

PhotoPrism is unable to access the "originals" directory, which typically stores original photos. 

PhotoPrism does start up properly, the webinteraface is accessible but no content is found in the originals folder.
Manually triggering the indexing process, using `photoprism stop`  or `photoprism start` does completely fail with the following error message:

```
root@photoprism:~# photoprism index
ERRO[2024-10-26T12:59:48+02:00] config: invalid originals path, check configuration and permissions
```

this might help:
systemctl is able to start and stop photoprism
chmod to 0777 all opt files did not help
/opt/photoprism/config/.env is read and used apparently but  `photoprism config` which is showing all used config options points to 
`config-path             | /root/.photoprism/storage/config`
the folder storage is not produced

### What settings are you currently utilizing?

Default Settings

### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

default paths in the config /opt/photoprism/config/.env:
```
PHOTOPRISM_STORAGE_PATH='/opt/photoprism/storage'
PHOTOPRISM_ORIGINALS_PATH='/opt/photoprism/photos/originals'
PHOTOPRISM_IMPORT_PATH='/opt/photoprism/photos/import'
```

### Please provide detailed steps to reproduce the issue.

- run the helper script with default settings 
- call `photoprism index` to trigger indexing
-> error

**Comment by tteck** [comment]
The Directories are there
https://github.com/tteck/Proxmox/blob/5ae25108efaf609f1ed30de38153034c357caaf6/install/photoprism-install.sh#L36
Here is the `.env` file that the script creates.
https://github.com/tteck/Proxmox/blob/5ae25108efaf609f1ed30de38153034c357caaf6/install/photoprism-install.sh#L44-L53
My focus is on developing application LXC scripts, and maintaining said scripts. I do not specialize in, nor do I have the time to provide support for operational issues. If you still insist your issue is directly related to the script, please explain
(within 1 hr ⌛), otherwise, I consider this closed.

**Comment by tteck** [comment]
🧐 As this issue is unrelated to the script, it is considered closed.

**Comment by NormannK** [comment]
sorry but that does not relate to the "ERRO[2024-10-26T12:59:48+02:00] config: invalid originals path, check configuration and permissions" after the default installation without changing anything. 
If anyone else finds this: I resolved it for myself by using the .deb Packages for Ubuntu / Debian in an LXC
https://web.photoprism.app/kb/installation-packages

**Comment by NormannK** [comment]
I found this error solved by someone else by accident: https://github.com/photoprism/photoprism/discussions/4247
tldr: `/opt/photoprism/config/.env`  is the wrong place to put the config. It should be in `/etc/photoprism`
I suggest creating the "defaults.yaml" there.
https://docs.photoprism.app/getting-started/config-files/defaults/

```
cat <<EOF >/etc/photoprism/defaults.yaml
ConfigPath: "~/.config/photoprism"
StoragePath: "~/.photoprism"
OriginalsPath: "~/Pictures"
ImportPath: "/media"
AdminUser: "admin"
AdminPassword: "helper-scripts.com"
AuthMode: "password"
DatabaseDriver: "sqlite"
HttpHost: "0.0.0.0"
HttpPort: 2342
HttpCompression: "gzip"
DisableTLS: false
DefaultTLS: true
Experimental: false
DisableWebDAV: false
DisableSettings: false
DisableTensorFlow: false
DisableFaces: false
DisableClassification: false
DisableVectors: false
DisableRaw: false
RawPresets: false
JpegQuality: 85
DetectNSFW: false
UploadNSFW: true
EOF
```

@tteck please take a look at that.
